### PR TITLE
Selecting Commerce Products as Related Assets

### PR DESCRIPTION
**Pull request by justinchoi001**

### CODE REVIEW

File: docs/commerce/2.x/en/creating-store-content/using-web-experience-management/selecting-commerce-products-as-related-assets.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Selecting Commerce Products as Related Assets
+
+Users can link to Commerce Products when creating Web Content articles. However, before Liferay Commerce Products are listed as [Related Assets](https://help.liferay.com/hc/articles/360028820532-Defining-Content-Relationships) in a [Web Content article](https://learn.liferay.com/dxp/7.x/en/content-authoring-and-management/web-content/user-guide/web-content-articles/adding-a-basic-web-content-article.html), users have configure the Liferay DXP Instance then execute a reindex.
+
+## Configure the DXP Instance
```
Comment (jrhoun) [inline docs/commerce/2.x/en/creating-store-content/using-web-experience-management/selecting-commerce-products-as-related-assets.md:5]: What is the person configuring though? The header here should be what the objective is - which I assume might be something like:

```
## Configure the Asset Browser to Search the Database
```

File: docs/commerce/2.x/en/creating-store-content/using-web-experience-management/selecting-commerce-products-as-related-assets.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+1. Navigate to the _Control Panel_ &rarr; _Configuration_ &rarr; _Search_.
+1. Click _Execute_ next to Reindex all search indexes.
+
+## Verify the Availability of Commerce Products
+
+Because Commerce Products are stored globally (that is, available on all Liferay DXP sites), users must change the Filter to _Global_.
+
+1. Click on _Site Administration_ &rarr; _Content_ &rarr; _Web Content_.
+1. Click the (![Add icon](../images/icon-add.png)) icon then _Basic Web Content_.
```
Comment (jrhoun) [inline docs/commerce/2.x/en/creating-store-content/using-web-experience-management/selecting-commerce-products-as-related-assets.md:25]: Icon link is broken.

File: docs/commerce/2.x/en/creating-store-content/using-web-experience-management/selecting-commerce-products-as-related-assets.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+1. Scroll down to the _Related Assets_ section.
+1. Click _Select_.
+1. Click _Commerce Product_.
+
+     ![Select Commerce Product in the Related Assets section.](./selecting-commerce-products-as-related-assets/images/01.png)
```
Comment (jrhoun) [inline docs/commerce/2.x/en/creating-store-content/using-web-experience-management/selecting-commerce-products-as-related-assets.md:30]: Image too grainy to be acceptable.

**Comment by jrhoun** [comment]
See my remarks and please re-send. Thank you.

File: docs/commerce/2.x/en/creating-store-content/using-web-experience-management/selecting-commerce-products-as-related-assets.md (hunk 2 above)
Comment (justinchoi001) [reply docs/commerce/2.x/en/creating-store-content/using-web-experience-management/selecting-commerce-products-as-related-assets.md:25]: Fixed

File: docs/commerce/2.x/en/creating-store-content/using-web-experience-management/selecting-commerce-products-as-related-assets.md (hunk 1 above)
Comment (justinchoi001) [reply docs/commerce/2.x/en/creating-store-content/using-web-experience-management/selecting-commerce-products-as-related-assets.md:5]: Renamed.

File: docs/commerce/2.x/en/creating-store-content/using-web-experience-management/selecting-commerce-products-as-related-assets.md (hunk 3 above)
Comment (justinchoi001) [reply docs/commerce/2.x/en/creating-store-content/using-web-experience-management/selecting-commerce-products-as-related-assets.md:30]: Updated new image

File: docs/commerce/2.x/en/creating-store-content/using-web-experience-management/selecting-commerce-products-as-related-assets.md (hunk 2 above)
Comment (justinchoi001) [reply docs/commerce/2.x/en/creating-store-content/using-web-experience-management/selecting-commerce-products-as-related-assets.md:25]: Fixed link

**Pull request closed without merging**